src/main/java/seedu/duke/data/ShoppingList.java

```diff
@@ -6,7 +6,7 @@
 
 public class ShoppingList {
 
-    private static ArrayList<Item> items = new ArrayList<>();
+    private ArrayList<Item> items = new ArrayList<>();
 
```

Comment:
Thanks for doing the housekeeping, I agree that this should not be static.

Reply:
Np! Actually I think GSON doesn't allow storing the items in the shopping list unless its non-static cause I had an issue with it that was only solved by changing this haha